cli/src/alluxio.org/cli/cmd/exec/test_hdfs_mount.go

```diff
@@ -0,0 +1,93 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package exec
+
+import (
+	"alluxio.org/cli/env"
+	"alluxio.org/log"
+	"bytes"
+	"fmt"
+	"github.com/palantir/stacktrace"
+	"github.com/spf13/cobra"
+)
+
+var TestHdfsMount = &TestHdfsMountCommand{
+	BaseJavaCommand: &env.BaseJavaCommand{
+		CommandName:   "testHdfsMount",
+		JavaClassName: "alluxio.cli.ValidateHdfsMount",
+		ShellJavaOpts: fmt.Sprintf(env.JavaOptFormat, env.ConfAlluxioLoggerType, "Console"),
+	},
+}
+
+type TestHdfsMountCommand struct {
+	*env.BaseJavaCommand
+	path     string
+	readonly bool
+	shared   bool
+	option   string
+}
+
+func (c *TestHdfsMountCommand) Base() *env.BaseJavaCommand {
+	return c.BaseJavaCommand
+}
+
+func (c *TestHdfsMountCommand) ToCommand() *cobra.Command {
+	cmd := c.Base().InitRunJavaClassCmd(&cobra.Command{
+		Use:   "testHdfsMount [--readonly] [--shared] [--option <key=val>] [--path <hdfsURI>]",
+		Short: "Tests runs a set of validations against the given hdfs path.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return c.Run(args)
+		},
+	})
+	cmd.Flags().StringVar(&c.path, "path", "",
```

Comment:
these types of flags can be marked as required and will fail if the user does not specify it
`cmd.MarkFlagAsRequired("path")` i think?
then can remove the `[required]` from the description

Reply:
Agree. Let me check how to throw errors / exceptions and implement it here.